### PR TITLE
Merge pull request #1 from syncrou/check_for_reqs

### PR DESCRIPTION
**Pull request by syncrou**
Added some additional docs
Added some requirements checking at the bash level
Added a bundler.require and Gemfile to manage gems.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'optimist'
```
Comment (lindgrenj6) [inline Gemfile:5]: Could we add `kubeclient` and `filewatcher` here as well?

Comment (syncrou) [reply Gemfile:5]: Yep, I'll get them added.

**Comment by syncrou** [comment]
@lindgrenj6 feedback applied if you could take a look.

**Review by lindgrenj6 — APPROVED** [review]
LGTM. Thanks Drew!

**Pull request merged**